pkgs/top-level/all-packages.nix

```diff
@@ -6251,6 +6251,8 @@ in
       pam = if stdenv.isLinux then pam else null;
     };
 
+  ssh-copy-id = openssh.ssh_copy_id;
```

Comment:
Note, that how nix-env works will ignore which output selected since. I think `nix profile install` may not have this same issue.

Reply:
Oh my, I wasn't aware of this. Running `nix-env -f . -A ssh-copy-id` does indeed link the whole ssh package into my user profile.

I think I'll change this PR to split out the ssh-copy-id package instead of using multiple outputs, since nix-env likely be here to stay for years to come.